force-app/main/default/classes/ResetDailyHabitTasks.cls

```diff
@@ -0,0 +1,61 @@
+//Reset the states of daily and habit task and add the resulted points to the employee
+global class ResetDailyHabitTasks implements Schedulable {
+    global void execute(SchedulableContext ctx) {
+        
+        try{
+             List<Contact> conts= new List<Contact>();//Not work ask why
+             List<Task_Monitor__c> toUpdateTasks= new List<Task_Monitor__c>();//Not work ask why
+           for(Contact cont : [SELECT Id, Points_Ammount__c, //Select contacts with daily and habit tasks
+                            	(Select 
+                                 State__c, Type__c, Habit_Complete__c,
+                                 Difficulty_val__c, Points_Awarded__c
+                                 from Tasks_Monitor__r where  Type__c<>'TO-DO') 
+                            FROM Contact where Id  IN (SELECT Contact__c FROM Task_Monitor__c 
+                             where  Type__c<>'TO-DO')])
+            {
+                 System.debug('Congtacto...................... ' + cont);
+                 System.debug('Found task');
+                //Calcule the points and update the 
+                for(integer i=0; i<cont.Tasks_Monitor__r.size(); i++){
+                   
+                    Task_Monitor__c task= cont.Tasks_Monitor__r[i];
+                    System.debug(task);
+                    if(task.Type__c=='Daily'){
```

Comment:
Its a good practice to always left spaces between the operators and criteria values. Please, keep it in mind

i.e:  task.Type__c == 'Daily',